EjercicioClases/Model/User/Buyer.pm

```diff
@@ -5,5 +5,13 @@ use warnings;
 use Data::Dumper;
 
 use Model::User;
+use Controller::DatabaseController;
 
 our @ISA = qw(Model::User);
+
+sub save{
+    my ($self, $buyer) = @_;
+    Controller::DatabaseController->save('buyers', $buyer);
```

Comment:
Here you can send $self and there is no need to require a $buyer as a parameter, you can refactor:
my ($self) = @_;
Controller::DatabaseController->save('buyers', $self);

And when you call the method:
$buyer->save();